src/main/resources/db/migration/common/V1.5.1_20230126_1500_addRegistrationDomainSelectionEnabled.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE configuration ADD COLUMN registration_domain_selection_enabled DEFAULT TRUE;
```

Comment:
Name of the file should mention version 1.5.2.
The SQL statements is missing the column type indication (please have a look at other migrations)